test/onnx/test_operators.py

```diff
@@ -816,7 +816,7 @@ def test_pixel_shuffle(self):
 
     def test_frobenius_norm(self):
         x = torch.randn(2, 3, 4).float()
-        self.assertONNX(lambda x: torch.norm(x, p="fro", dim=(0, 1), keepdim=True), x)
+        self.assertONNX(lambda x: torch.linalg.norm(x, p="fro", dim=(0, 1), keepdim=True), x)
```

Comment:
The new api uses `ord` instead of the `p` kwarg, to match numpy.

Reply:
done for 3 files